RevenueCatUI/Helpers/PreviewHelpers.swift

```diff
@@ -73,7 +76,7 @@ struct PreviewableTemplate<T: TemplateViewType>: View {
     }
 
     var body: some View {
-        if self.presentInSheet {
+        if self.presentInSheet || self.interfaceIdiom == .pad {
```

Comment:
`PaywallView` is expected to be displayed in a sheet in iPad instead of fullscreen. This helps previewing paywalls on the iPad simulator with the right proportions.